GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -1025,6 +1025,14 @@ public static ArgumentString FindRenamesAndCopiesOpts()
             return FindRenamesOpt() + findCopies;
         }
 
+        public static ArgumentString LsRemotesCmd(string url)
```

Comment:
This has a single use, I think we can inline it at the callsite until we have more uses for it.
Besides now we need to all sort of validations here, which makes it more complex than required